dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -272,6 +388,62 @@ RSResult MinMaxIndex::checkGreater(size_t pack_index, const Field & value, const
     }
     return RSResult::Some;
 }
+
+RSResult MinMaxIndex::checkNullableGreaterEqual(size_t pack_index, const Field & value, const DataTypePtr & type)
+{
+    const ColumnNullable & column_nullable = static_cast<const ColumnNullable &>(*minmaxes);
+
+    const auto * raw_type = type.get();
```

Comment:
ditto

Reply:
> ditto

Caller has checked the type with typeid_cast<const DataTypeNullable *> and extracted the nested type with removeNullable(type).